### DISCUSSION TITLE
Trying to boot from USB - get grub cmdline

### DISCUSSION THREAD

**Discussion by scheenkbgates** [Security-Onion-Solutions/securityonion#5895] [Unsupported Versions] [answered]
Hi,
Not sure why this is happening and need help. I have read similar issues but no resolutions.
I have downloaded the iso and put it on a USB stick using rufus, tried balena etcher but that doesnt work.

Problem: When the server begins to detect the usb, it loads and then eventually lands on cmd line with grub> 

Similar issues from others have said that putting the BIOS to legacy or CSM will fix this, except that the ISO file cannot be used in that mode, the only option for the ISO in rufus is UEFI.

Booting the server in legacy mode regardless of that, will of course come up with a page saying "This needs to be booted in UEFI mode" Then as stated earlier when I boot in UEFI mode it goes straight to the grub cmd line.

The server I am using is a PowerEdge R620.

**Comment by dougburks** [comment, marked as answer]
Have you tried the steps at https://docs.securityonion.net/en/2.3/trouble-booting.html?

**Reply by scheenkbgates** [reply]
Thanks Doug, I made sure all of that was correct, but still it would not boot and instead would just go to grub menu.

I tried using Balena Etcher again and was able to get that to work over Rufus, and this time the boot worked on the server!

So the Resolution here I would say is, that Rufus didnt correctly put the iso on my USB, or the USB didnt like Rufus (even though it would successfully put the iso on). Either way the boot worked by setting my BIOS back to BIOS/Legacy from UEFI. 

Thanks.
